lib/dialyzer/src/typer_core.erl

```diff
@@ -0,0 +1,1040 @@
+%% -*- erlang-indent-level: 2 -*-
```

Comment:
Since this is a new file, can you change tabs to spaces (still 8 spaces per tab)?

Reply:
How would you do that? I'm not an emacs user, I just copied what was in the other file.

Reply:
Oh, I meant just replacing all existing tabs with 8 spaces. Our [.editorconfig](https://github.com/erlang/otp/blob/master/.editorconfig) should take care of the rest. :)

Reply:
Ohhh… I didn't want to do that in case you want to compare this code with the previous version of `typer.erl`.
If it's up to us, we'll then apply `rebar3_format` to it, of course.

Reply:
In other words: The question is… _Should we preserve as much of `typer.erl`'s structure/text/etc. as possible or what are the rules on how to format new modules in this project?_

Reply:
> In other words: The question is… _Should we preserve as much of `typer.erl`'s structure/text/etc. as possible or what are the rules on how to format new modules in this project?_

The general rule is to not disturb the current formatting if one exists, and that the Emacs `erlang-mode` style should be used for new files like this one.

Reply:
So just turn tabs into 8 spaces, then… ok.

Reply:
Done ☑️